python/cudf/requirements/requirements-build.txt

```
@@ -0,0 +1,25 @@
+cmake
+wheel==0.34.2
+cmake-setuptools>=0.1.3
+numpy
+pandas>=1.0,<1.3.0dev0
+cython>=0.29,<0.30
+sphinx
+sphinx_rtd_theme
+sphinxcontrib-websupport
+nbsphinx
+numpydoc
+recommonmark
+pandoc==2.0a4
+mypy==0.782
+typing_extensions
+dlpack
+rapidjson
+flatbuffers
+sphinx-markdown-tables
+sphinx-copybutton
+packaging
+protobuf
+nvtx>=0.2.1
+cachetools
+numba>=0.49.0,!=0.51.0
```

I think many of these can be removed as they're only used for development for docs / linting / etc.
```suggestion
numpy
pandas>=1.0,<1.3.0dev0
cython>=0.29,<0.30
typing_extension
flatbuffers
packaging
protobuf
nvtx>=0.2.1
cachetools
numba>=0.49.0,!=0.51.0
```